dev/benchmark/sysbench-in-k8s.md

```diff
@@ -0,0 +1,460 @@
+| 150     | 60732.84  | 69.29           |
+| 300     | 91005.98  | 90.78           |
+| 600     | 110517.67 | 167.44          |
+| 900     | 119866.38 | 235.74          |
+| 1200    | 125615.89 | 282.25          |
+| 1500    | 128501.34 | 344.082         |
+
+![OLTP Read Write](/media/sysbench-in-k8s/oltp-read-write-qps.png)
+![OLTP Read Write](/media/sysbench-in-k8s/oltp-read-write-latency.png)
+
+### Performance comparison between single AZ and multiple AZs
+
+The network latency on communication across multiple AZs in GCP is slightly higher than that within the same zone. In this test, machines of the same configuration are used in different deployment plans under the same standard. The purpose is to learn how the latency across multiple AZs might affect the performance of TiDB.
+
+Single AZ:
+
+| Threads | QPS       | 95% latency(ms) |
+| :------ | :-------- | :-------------- |
+| 150     | 203879.49 | 1.37            |
+| 300     | 272175.71 | 2.30            |
+| 600     | 287805.13 | 4.10            |
+| 900     | 295871.31 | 6.21            |
+| 1200    | 294765.83 | 8.43            |
+| 1500    | 298619.31 | 10.27           |
+
+Multiple AZs:
+
+| Threads | QPS       | 95% latency(ms) |
+| :------ | :-------- | :-------------- |
+| 150     | 141027.10 | 1.93            |
+| 300     | 220205.85 | 2.91            |
+| 600     | 250464.34 | 5.47            |
+| 900     | 257717.41 | 7.70            |
+| 1200    | 258835.24 | 10.09           |
+| 1500    | 280114.00 | 12.75           |
+
+QPS comparison:
+
+![Single Zonal vs Regional](/media/sysbench-in-k8s/single-zonal-vs-regional-qps.png)
+
+Latency comparison:
+
+![Single Zonal vs Regional](/media/sysbench-in-k8s/single-zonal-vs-regional-latency.png)
+
+From the images above, the impact of network latency goes down as the concurrency pressure increases. In this situation, the extra network latency is no longer the main bottleneck of performance.
+
+## Conclusion
+
+This is a test of TiDB using sysbench running in Kubernetes deployed on a typical public cloud platform. The purpose is to learn how different factors might affect the performance of TiDB. On the whole, these influencing factors include the following items:
+
+- In the VPC-Native mode, TiDB performs slightly better in Host network than in Pod network. (The difference, ~7%, is measured in QPS. Performance differences caused by the factors below are also measured by QPS.)
+- In Host network, TiDB performs better (~9%) in the read test on Ubuntu provided by GCP than on COS.
+- The TiDB performance is slightly lower (~5%) if it is accessed outside the cluster via Load Balancer.
+- Increased latency among nodes in multiple AZs has certain impact on the TiDB performance (30% ~ 6%. The impact diminishes )
+- The QPS performance is greatly improved (50%-60%) if the Point Select read test is conducted on machines of computing type (compared with general types), because the test mainly consumes CPU resources.
```

Comment:
```suggestion
- The QPS performance is greatly improved (50% ~ 60%) if the Point Select read test is conducted on machines of computing type (compared with general types), because the test mainly consumes CPU resources.
```